openedx/adg/lms/webinars/tests/test_helpers.py

```diff
@@ -197,3 +204,71 @@ def test_remove_emails_duplicate_in_other_list(emails, reference_emails, expecte
     Test that only the list of emails that are not present in reference list of emails are returned
     """
     assert sorted(remove_emails_duplicate_in_other_list(emails, reference_emails)) == sorted(expected_emails)
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    'template, msg_id_field_name', [
+        (MandrillClient.WEBINAR_TWO_HOURS_REMINDER, STARTING_SOON_REMINDER_ID_FIELD_NAME),
+        (MandrillClient.WEBINAR_ONE_WEEK_REMINDER, ONE_WEEK_REMINDER_ID_FIELD_NAME),
+    ]
+)
+def test_save_scheduled_reminder_ids(template, msg_id_field_name):
+    """
+    Tests save_scheduled_reminder_ids method is saving reminder ids properly.
+    """
+    registration = WebinarRegistrationFactory()
```

Comment:
Since it is used thrice, we can create a fixture for it.

Reply:
We create factories so that we can use them whenever we need to create an object. But for a single object I don't think it is a good idea to create a fixture.

Reply:
My concern is code duplication. `registration = WebinarRegistrationFactory()` is used thrice. I think if using a fixture will eliminate duplication, then we should. Also, there is already a fixture in place for Webinar, so we are already using the practice of fixtures for such a case in the project.

Any particular drawbacks in your mind related to using a fixture in this case?